api/utils/update_user.py

```diff
@@ -1,35 +1,37 @@
 from firebase_admin import auth
-from flask import jsonify, make_response
-from werkzeug.security import generate_password_hash
 
-from api.utils.errors import ErrorResponse
-from api.models.user import User
-from api.schemas.errors import OperationNotFound
 
-
-def update_user(uid, password, photoURL, username):
+def update_firebase_photoURL(uid, photoURL):
     try:
-        user = auth.update_user(
+        auth.update_user(
             uid=uid,
-            password=password,
-            username=username,
             photoURL=photoURL
         )
-    except Exception:
-        return ErrorResponse(OperationNotFound().message, 422, {'Content-Type': 'application/json'}).respond()
+        return True
+    except Exception as e:
+        print(e)
```

Comment:
Just asking, are these for server logging or were these debug statements which your forgot to remove?

Reply:
These are for server logging. This way we  will be able to log the cause of error in case of exception.